src/EA.Weee.Integration.Tests/Handlers/GetSchemesWithObligationHandlerTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace EA.Weee.Integration.Tests.Handlers
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Autofac;
+    using AutoFixture;
+    using Base;
+    using Builders;
+    using Core.Scheme;
+    using Core.Shared;
+    using Domain.Scheme;
+    using FluentAssertions;
+    using NUnit.Specifications;
+    using Prsd.Core;
+    using Prsd.Core.Autofac;
+    using Prsd.Core.Mediator;
+    using Requests.Admin.Obligations;
+    using SchemeStatus = Domain.Scheme.SchemeStatus;
+
+    public class GetSchemesWithObligationHandlerTests : IntegrationTestBase
```

Comment:
can you update the file name and class name as adding **integration** to them?

Reply:
done...